frontend/src/app/modules/settings/user/user-profile-page/user-profile-page.component.html

```diff
@@ -93,7 +93,7 @@
             <ng-moment-timezone-picker
                 [formControlName]="'timeZone'"
                 customPlaceholderText=""
-                [getUserZone]="!user?.timeZone">
+                [getUserZone]="!user.timeZone">
```

Comment:
I think it was a fix for bug of broken form time-zone combo-box when page are reloading (Valko Eughen's fix)